webpack/webpack.config.dev-server.js

```diff
@@ -65,6 +65,7 @@ module.exports = {
         new webpack.DefinePlugin({
           __DEVCLIENT__: false,
           __DEVSERVER__: true
-        })
+        }),
+      new webpack.IgnorePlugin(/vertx/)
```

Comment:
I didn't get any errors after adding it to the dev-server. You mean it should go into "webpack.config.prod.js" as well?